payments/providers/cpu_ceepos.py

```diff
@@ -0,0 +1,360 @@
+import hashlib
+import hmac
+import logging
+
+import requests
+from django.http import HttpResponse
+from requests.exceptions import RequestException
+
+from ..exceptions import (
+    DuplicateOrderError,
+    OrderStateTransitionError,
+    PayloadValidationError,
+    PaymentCreationFailedError,
+    ServiceUnavailableError,
+    UnknownReturnCodeError,
+)
+from ..models import Order, OrderLine
+from ..utils import price_as_sub_units
+from .base import PaymentProvider
+
+logger = logging.getLogger(__name__)
```

Comment:
I don't remember if respa has specific conventions about the loggers, but I find it clearer if the module level logger variable is named with capitals, usually simply `LOG`. That way (a) it's easy to spot the logging calls and (b) it's easier to figure out that it's probably not a local variable, but a global one, when reading the code of a function that uses the logger.

Reply:
I changed the variable name to `LOG`.